x-pack/legacy/plugins/monitoring/public/components/chart/_chart.scss

```diff
@@ -4,6 +4,15 @@
   -webkit-tap-highlight-color: transparent;
 }
 
+.monRhythmChart__wrapper .monRhythmChart__zoom {
+  visibility: hidden;
+  padding-right: 12px;
```

Comment:
Please use an EUI SASS variable instead of a hardcoded pixel value. In this case, you would want `$euiSizeM`. For future reference, the available size variables can be found here: https://github.com/elastic/eui/blob/master/src/global_styling/variables/_size.scss